xonsh/proc.py

```diff
@@ -15,7 +15,7 @@
 from functools import wraps
 from threading import Thread
 from collections import Sequence, namedtuple
-from subprocess import Popen, PIPE, DEVNULL, STDOUT, TimeoutExpired
+from subprocess import Popen, PIPE, DEVNULL, STDOUT, TimeoutExpired, CalledProcessError
```

Comment:
It looks like this change pushed this over the PEP8 limit of 80 characters per line.  This can be refactored by using `from subprocess import (...)` and making the "tuple" span multiple lines.